CMakeModules/CPackConfig.cmake

```diff
@@ -101,5 +101,9 @@ SET(CPACK_SOURCE_IGNORE_FILES
 # Ignore build directories that may be in the source tree
 FILE(GLOB_RECURSE CACHES "${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 
+SET(CPACK_WIX_LICENSE_RTF "${PROJECT_SOURCE_DIR}/LICENSE.rtf")
+SET(CPACK_WIX_PATCH_FILE "${PROJECT_SOURCE_DIR}/wix/WIXPatch.wxs")
+SET(CPACK_WIX_UPGRADE_GUID "FF9E2D77-CDC7-4D24-8B7B-99D66EDEE862")
```

Comment:
@InBetweenNames I was wondering if this is randomly generated. If so does it need to change with every version ?

Can you also add some comments on when this string needs to be preserved and when it needs to be regenerated ?